SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Vulnerability Disclosure and Response Process
+
+As we all know, TiDB is a fast-growing open source database. In order to ensure its security, we have adopted the following security vulnerability disclosure and response process.
+
+The primary goal of this process is to reduce users’ total exposure time to publicly known vulnerabilities. To ensure TiDB products’ vulnerabilities can be fixed quickly, our security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
+
+If you think you have found a vulnerability or have experienced security incidents involving vulnerabilities of TiDB products, please report it as soon as possible to the TiDB security team (security@tidb.io).
+
+Please kindly help provide as much vulnerability information as possible in the following format:
+
+- Issue title*:
+
+- Overview*:
+
+- Affected components and version number*:
+
+- CVE number (if any):
+
+- Vulnerability verification process*:
+
+- Contact information*:
+
+*mean required
+
+# Response time
+
+The TiDB security team will confirm the vulnerabilities within 2 working days after your submission and keep in touch with you.
+
+After a security vulnerability is fixed, we will thank the reporter publicly. However, in order to avoid negative impact, please kindly keep the vulnerability confidential before that. We would be really appreciated if you could obey the following code of conduct:
```

Comment:
```suggestion
After we fix the informed security vulnerability, we will thank you publicly. However, to avoid unnecessary troubles, keep the vulnerability confidential until we fix it. We would appreciate it if you could obey the following code of conduct:
```

Reply:
```suggestion
We will publicly thank you after fixing the informed security vulnerabilities. To avoid unnecessary troubles, please keep the vulnerability confidential until we fix it. We would appreciate it if you could obey the following code of conduct:
```